source/_integrations/zwave_js.markdown

```diff
@@ -1093,21 +1094,21 @@ Set the log level for `zwave_js_server` to a level higher than `debug`. This can
 
 This sections lists functionality that is available in Z-Wave but that is not currently supported in Home Assistant.
 
-### Setting the controller into learn mode to receive network information
+### Setting the adapter into learn mode to receive network information
 
-In Home Assistant, it is currently not possible to set the Z-Wave controller into learn mode to receive network information from another controller.
+In Home Assistant, it is currently not possible to set the Z-Wave adapter into learn mode to receive network information from another adapter.
 
-### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
+### Including / excluding a adapter in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
```

Comment:
```suggestion
A Z-Wave adapter that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave adapter to join another network, because Home Assistant acts as the central hub.
```
On the same note, one might argue that the first and third instance here refers to the hardware? Maybe?